scripts/manager_char_4e_tmt.lua

```diff
@@ -0,0 +1,74 @@
+-- 
+-- Please see the license.txt file included with this distribution for 
+-- attribution and copyright information.
+--
+
+rPowerContainsDataCondition = nil;
+rPowerUsedEvent = {
+    sName = "power_used_event",
+    aParameters = { 
+        "rSource",
+        "rPower"
+    };
+};
+
+function onInit()
+    TriggerManager.defineEvent(rPowerUsedEvent);
+    initializeConditions();
+end
+
+function initializeConditions()
+    rPowerContainsDataCondition = {
+        sName = "power_used_contains_data",
+        fCondition  = powerContainsDataCondition,
+        aRequiredParameters = {
+            "rSource",
+            "rPower"
+        },
+        aConfigurableParameters = {
+            {
+                sName = "sPowerProperty",
+                sDisplay = "power_used_property_parameter",
+                sType = "combo",
+                aDefinedValues = {
+                    "power_used_property_name",
+                    "power_used_property_keywords",
+                    "power_used_property_recharge",
+                    "power_used_property_action",
+                    "power_used_property_range",
+                }
+            },
+            {
+                sName = "sPowerContains",
```

Comment:
This is perfectly usable as is and the below ramblings of mad man should not be brought into current scope....

What if there was also a combobox that was used instead and dynamically populated for the properties with discrete value sets!

Reply:
At least for 4e, there's dozens of possibilities for keywords alone, about a dozen different actions, half a dozen ranges. That might be a bit much, if I'm understanding the ramblings correctly.

Reply:
Maybe an autofill then if fancy moon talk is happening. The base idea is that if there is a "correct" bit of text I'd like to keep my hammy little fingers away from mistyping it. Dealing narcotic damage lends a rather different vibe to an encounter after all.